### ISSUE TITLE
Précisions sur le modèle viable minimal

### ISSUE THREAD

**Issue by erispoe** [EPFL-AgentBased-2014/Cours#35]
Bonjour à tous,

Certains groupes ont commencé à écrire du code pour le modèle viable minimal. Nous ne vous empêchons pas de prendre un peu d'avance en codant, mais l'exercice du modèle viable minimal ne correspond pas à cela, il est beaucoup plus simple. Il s'agit de sélectionner, parmi tous les éléments possibles de votre modèle, ceux qui sont absolument essentiels pour avoir une première version fonctionnelle (viable) de votre modèle dans Netlogo. Ce sont ces éléments que vous pourrez commencer à implémenter dans le code de votre modèle en début de semaine prochaine.

Par exemple, vous pouvez vouloir tester l'influence des paramètres A, B, C, D et E sur l'évolution d'une population P. Peut-être que les paramètres B et D vous permettent de construire une premier modèle fonctionnel que vous pourrez ensuite compléter/complexifier.

L'exercice du modèle viable minimal est celui d'identifier les éléments dont vous avez absolument besoin pour ce premier modèle fonctionnel, qui est certainement plus simple ou moins complet que le modèle idéal que vous pouvez avoir en tête. Bien sûr, il y a parfois plusieurs possibilités et vous devez faire un choix, par où commencer.

Vous pouvez poster vos questions ou remarques en commentaire de cette _issue_.

---

@guerber @judoka @rothanita @davidkilic @kvos @ThomasSteyaert @Matthferr @benustucki @Jenacl @Konde @hugomazza @tlgaglia @Lulubec 
@hadumi @rufener @thampipraveen @RosenfeldAlois @Sbfasel @gabzouz @lbornet @Jolyjumper @jrausis @liebich @samifarra @bmrapit @HotelBravo @pierrenac 

cc @aourednik @3arbouch @miryamch @Geheiner

**Comment by Lulubec** [comment]
En fait on doit faire un modèle commun pour tout le groupe ou chacun doit faire un modèle différent autour d'un thème commun?

**Comment by RosenfeldAlois** [comment]
Hello!
D'après ce que j'ai compris, c'est un modèle commun qui va être développé par tout le groupe.

**Comment by erispoe** [comment]
Bonsoir, 

Oui, c'est cela. Il s'agit de poser les bases d'un modèle commun pour le groupe, éventuellement en vous inspirant ou en réutilisant des éléments de vos propositions individuelles, qui ont servies à vous regrouper par affinités.

**Comment by LauraIten** [comment]
Quand j'ai chargé un fichier sur GitHub dans mon groupe, comment est-ce que je fais pour que les autres du groupe puissent le voir? Jusqu'à ce moment c'est que moi qui voit ce fichier...

**Comment by erispoe** [comment]
Oui, vous avez créé une copie du dépôt du groupe (un fork), sur votre compte personnel. Il faut soumettre le fichier au dépôt du groupe directement: https://github.com/EPFL-AgentBased-2014/Groupe9 et non https://github.com/LauraIten/Groupe9

PS: si vous avez un problème différent de celui qui est discuté dans l'issue, vous devriez ouvrir une nouvelle issue avec un titre explicite.
